crate/operator/create.py

```diff
@@ -1009,6 +1010,98 @@ async def create_services(
         )
 
 
+async def create_data_service(
```

Comment:
Since `create_services()` is idempotent, I'm not sure we need to split this out - you could just call `create_services()` and it will re-create the LoadBalancer and not touch the discovery service.